src/voice/voice-quota.service.ts

```
@@ -16,41 +16,38 @@ export class VoiceQuotaService {
     }
 
     async checkUsage(userId: string): Promise<boolean> {
-        // 1. Get User Subscription Status
-        // Quick check on user's role or plan
+        const currentMonth = this.getCurrentMonth();
```

ESLint

[error] Delete `····`

(prettier/prettier)